app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ChargesController < ApplicationController
+
+  def new
+  end
+
+  def create
+
+    if session[:cart_id]
+      @cart = Cart.find(session[:cart_id])
+    end
+
+    # Amount in cents
```

Comment:
Do we need this info here? Maybe it could be an idea to put it in the view (or in a alert message) so the customer understand that he/she need to specify the amount in cent, IF required (don't know how the gem works yet).